syft/formats/common/cyclonedxhelpers/licenses.go

```diff
@@ -3,51 +3,41 @@ package cyclonedxhelpers
 import (
 	"github.com/CycloneDX/cyclonedx-go"
 
-	"github.com/anchore/syft/internal/spdxlicense"
 	"github.com/anchore/syft/syft/pkg"
 )
 
+// TODO: update this to only return valid cyclonedx expression types
+// This should be a function that just surfaces licenses already validated in the package struct
 func encodeLicenses(p pkg.Package) *cyclonedx.Licenses {
 	lc := cyclonedx.Licenses{}
-	for _, licenseName := range p.Licenses {
-		if value, exists := spdxlicense.ID(licenseName); exists {
+	for _, l := range p.Licenses {
+		if l.SPDXExpression != "" {
```

Comment:
There needs to be a way to detect if this is a singular license or a full expression by using either `License` or `Expression` on the `LicenseChoice` struct (not `ID` vs `Name` on the same `License` struct).